spec/grid_view_spec.rb

```diff
@@ -1,9 +1,11 @@
+require_relative "../lib/grid"
 require_relative "../lib/grid_view"
 
 RSpec.describe GridView do
   it "renders a grid to output" do
     output = StringIO.new
-    view = described_class.new(output: output)
+    grid = Grid.new("B2" => "X", "C3" => "O")
+    view = GridView.new(grid: grid, output: output)
```

Comment:
This is a cool spec. Not sure why I like it so much, but it just is so easy to understand. You put data in and you get out a grid that makes sense.